### PR TITLE
En 10597 prepare block result

### PR DESCRIPTION
**Pull request by mariusmihaic**

### CODE REVIEW

File: process/utility/utils.go [hunk 1]
```diff
@@ -1,7 +1,9 @@
 package utility
 
 import (
+	"bytes"
```
Comment (miiu96) [inline process/utility/utils.go:4]: sort imports

File: process/utility/utils_test.go [hunk 2]
```diff
@@ -2,6 +2,8 @@ package utility_test
 
 import (
 	"github.com/ElrondNetwork/covalent-indexer-go/process/utility"
+	"github.com/ElrondNetwork/covalent-indexer-go/schema"
```
Comment (miiu96) [inline process/utility/utils_test.go:5]: sort imports

File: testscommon/data_generator.go [hunk 3]
```diff
@@ -6,6 +6,16 @@ import (
 	"strconv"
 )
 
+func GenerateRandomFixedBytes(n int) []byte {
```
Comment (miiu96) [inline testscommon/data_generator.go:9]: add a comment for the exported function

File: testscommon/data_generator.go [hunk 4]
```diff
@@ -6,6 +6,16 @@ import (
...
+	ret := make([]byte, n)
+
+	for i := 0; i < n; i++ {
+		ret[i] = byte(rand.Int())
+	}
+
+	return ret
+}
+
 func GenerateRandomBytes() []byte {
```
Comment (miiu96) [inline testscommon/data_generator.go:19]: add a comment for the exported function

**Review by AdoAdoAdo — APPROVED** [review]
review findings will be done on EN-10606-Review_fixes

**Review by miiu96 — APPROVED** [review]

**Review by AdoAdoAdo — COMMENTED** [review]
fixes to be done on EN-10606-Review_fixes

File: process/utility/utils.go (hunk 1 above)
Comment (mariusmihaic) [reply process/utility/utils.go:4]: Done

File: process/utility/utils_test.go (hunk 2 above)
Comment (mariusmihaic) [reply process/utility/utils_test.go:5]: Done

File: testscommon/data_generator.go (hunk 3 above)
Comment (mariusmihaic) [reply testscommon/data_generator.go:9]: Done

File: testscommon/data_generator.go (hunk 4 above)
Comment (mariusmihaic) [reply testscommon/data_generator.go:19]: Done

**Pull request merged**